### PR TITLE
Improved ActiveRecord Connection Pool docs [ci skip]

### PR DESCRIPTION
**Pull request by amitsuroliya**

### CODE REVIEW

File: activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module ActiveRecord
   class ConnectionTimeoutError < ConnectionNotEstablished
   end
 
-  # Raised when a pool was unable to get ahold of all its connections
+  # Raised when a pool was unable to get a hold of all its connections
```
Comment (kaspth) [inline activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb]: The original was correct. Can you change it back? :grin:

File: activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb [hunk 2]
```diff
@@ -638,7 +638,7 @@ def attempt_to_checkout_all_existing_connections(raise_on_acquisition_timeout =
           raise
         end
       rescue Exception # if something else went wrong
-        # this can't be a "naked" rescue, because we have should return conns
+        # this can't be a "naked" rescue, because we should have return conns
```
Comment (kaspth) [inline activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb]: This isn't public documentation but internal comments, I'd prefer to keep the git history here — can you change it back? :grin:

**Comment by amitsuroliya** [comment]
@kaspth Updated PR. Thanks for review :heart:

**Comment by kaspth** [comment]
Thanks!

**Pull request merged**